examples/calico-cert.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: "giantswarm.io/v1"
+kind: Certificate
+metadata:
+  name: "uo91f-calico"
+  namespace: "uo91f"
+spec:
+  clusterID: "uo91f"
+  clusterComponent: "calico"
+  commonName: "calico.uo91f.g8s.eu-west-1.aws.test.private.giantswarm.io"
```

Comment:
Is the calico domain correct? It isn't in the clustertpr, should we add it?

https://github.com/giantswarm/clustertpr/blob/master/calico/calico.go